add-missing-campaign.js

```diff
@@ -0,0 +1,207 @@
+export const addMissingCampaign = async (client) => {
+
+    const createRewardPromotionObject = (firstValidationRule, secondValidationRule, thirdValidationRule) => {
```

Comment:
We do not have typescript support here, so even more important is to pay attention to naming. Please note that the `createRewardPromotionObject` function accepts as parameters id of three validation rules, not validation rule definitions from below. Therefore, the `firstValidationRule` name has two meanings in that file, once its rule id, and once its rule definition. I suggest changing a name from `firstValidationRule` to `firstValidationRuleId`